dynawo/sources/Common/DYNSparseMatrix.cpp

```diff
@@ -66,7 +66,7 @@ SparseMatrix::changeCol() {
 }
 
 void
-SparseMatrix::addTerm(const int& row, const double& val) {
+SparseMatrix::addTerm(const int row, const double val) {
```

Comment:
Those unrelated modifications make it harder to see the core of your modifications. Would it be possible to put them in another PR?

Reply:
I split it in two so review can be done only on the interesting part

Reply:
ok thanks. However no need to worry before we answer my concern in the first message as it might be a showstopper.